reactioncommerce/reaction
# feat: new button component with isWaiting loading state

Impact: **minor**  
Type: **feature**

## Issue
Wrap MUI's button component to allow for loading states.

## Testing
1. `import Button from "/imports/client/ui/components/Button";`
1. set `isWaiting` to true
1. set `onClick={() => alert("test")`
1. see loading state
1. see everything else works as it should
